upsint/cli.py

```diff
@@ -192,13 +196,48 @@ def update_labels(source_repo, service, source_service, destination):
         ))
 
 
+@click.command(name="remove-merged-branches")
+@click.argument('merged_with_branch', type=click.STRING, default="master")
+def remove_merged_branches(merged_with_branch):
+    """
+    Remove branches which are already merged (in master by default)
+
+    Argument MERGED_WITH_BRANCH defaults to master and checks whether
+    a branch was merged with this one
+    """
+    a = App()
+    to_remove = []
+    for branch_dict in a.list_branches(merged_with_branch):
+        branch_name = branch_dict["name"]
+        if branch_name == merged_with_branch or branch_name == branch_dict["remote_tracking"]:
+            # don't remove self or a local copy of remote branch
+            continue
+        if branch_dict["merged"] == "merged":
+            to_remove.append(branch_name)
+    if not to_remove:
+        print("Nothing to remove.")
+        return
+    print("Shall we remove these local branches?")
+    for b in to_remove:
+        print(f"* {b}")
+    inp = input("Y/N? ")
+    if inp in ("y", "Y", "yolo"):
```

Comment:
It is angry with me when I answer (Y or N):

```
[dhodovsk@shiny packit-tmp]$ upsint remove-merged-branches
Shall we remove these local branches?
* version-compare
* flask
Y/N? Y
Traceback (most recent call last):
  File "/usr/local/bin/upsint", line 11, in <module>
    load_entry_point('upsint==0.1.dev72+g42ebe9b', 'console_scripts', 'upsint')()
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/upsint/cli.py", line 224, in remove_merged_branches
    import ipdb; ipdb.set_trace()
ModuleNotFoundError: No module named 'ipdb'
```

Reply:
Yeah, I forgot to remove debugger -_-

Reply:
Oh, I pushed it last week, please fetch latest commits.